app/views/admin/invoices/show.html.erb

```diff
@@ -1,7 +1,20 @@
 <h3>Invoice <%=@invoice.id %></h3>
-
+<br>
 <div id="invoice-details">
   <p>Date Created: <%= @invoice.formatted_date %></p>
   <p>Customer Name: <%= @invoice.customer_name %></p>
   <p>Status: <%= @invoice.status %></p>
-</div>
\ No newline at end of file
+</div>
+<br>
+<div id="invoice_items-details">
```

Comment:
Some of the HTML formatting can probably be changed, so that it is going to be in a table format. 

All the information is there though.